ouroboros-network/src/Ouroboros/Network/BlockFetch/Decision.hs

```diff
@@ -71,6 +72,10 @@ data FetchDecisionPolicy header = FetchDecisionPolicy {
        blockFetchSize          :: header -> SizeInBytes
      }
 
+-- | The current syncing mode that we are in
```

Comment:
Shouldn't we describe these states, what they mean, and give a rough idea about what causes transitions between them?